src/lib/y2storage/devicegraph.rb

```diff
@@ -119,11 +119,23 @@ def actiongraph(from: nil)
       Actiongraph.new(graph)
     end
 
+    # @return [Array<Dasd>]
+    def dasds
+      Dasd.all(self)
+    end
+
     # @return [Array<Disk>]
     def disks
       Disk.all(self)
     end
 
+    # All disk devices. This method is necessary because DASD devices
+    #   are not returned when asking for disks.
+    # @return [Array<Dasd, Disk>]
+    def disk_devices
+      dasds + disks
```

Comment:
I am not sure, but I would say so, or at least it is a reasonable option. When we have dasd disks, the system should be installed in them, but I do not know if this is a good criteria to be first in the list. Another option would be to order them by name, for example.